pages/osx/brew-mas.md

```diff
@@ -0,0 +1,32 @@
+# brew mas
+
+> [mas-cli](https://github.com/mas-cli/mas) is a simple command line interface for the Mac App Store. Designed for scripting and automation.
+
+- Search the Mac App Store by app name and return matching identifiers:
+
+`mas search {{app_name}}`
+
+- Install or update an application, not for the first time (apps must already be in the Purchased tab of the App Store):
+
+`mas install {{app_name}} {{app_identifier}}`
+
+- Show all installed applications and their product identifiers:
+
+`mas list`
+
+- List installed apps with pending updates.
+
+`mas outdated`
+
+- Install all pending updates:
+
+`mas upgrade`
+
+- Install updates for a specific app:
+
+`mas upgrade {{app_identifier}}`
+
+- Reset mas:
+
+`mas reset`
+
```

Comment:
Looks like there's also some trailing whitespace at the bottom which is being picked up by the linter.